### PR TITLE
PHR-11742 don't send letter invitations to under-16s

### PR DESCRIPTION
**Pull request by jimmymcfox**

### CODE REVIEW

**Review by pkbdevdeploy — COMMENTED** [review]
Version `66-head-d719c47-338776-SNAPSHOT` was generated from commit `d719c47b1235acbba619c8e1136f7eabd8101759`

File: util/pom.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <groupId>com.pkb.pkbcommon</groupId>
+        <version>${revision}${changelist}</version>
+    </parent>
+
+    <artifactId>util</artifactId>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (idasbiste) [inline util/pom.xml:17]: Do we specifically need these?

**Review by idasbiste — CHANGES_REQUESTED** [review]
The method looks good but I would probably rebase this as it is dragging a lot of old commits. Also, I have left a question about possible unnecessary properties in the util pom.

**Review by pkbdevdeploy — COMMENTED** [review]
Version `66-head-50c32dd-339153-SNAPSHOT` was generated from commit `50c32dd0321663771f6f3313365c68da83e230ca`

**Comment by jimmymcfox** [comment]
Created new branch, will raise new PR

**Pull request closed without merging**